addons/account/models/chart_template.py

```diff
@@ -109,8 +109,8 @@ class AccountChartTemplate(models.Model):
     account_ids = fields.One2many('account.account.template', 'chart_template_id', string='Associated Account Templates')
     tax_template_ids = fields.One2many('account.tax.template', 'chart_template_id', string='Tax Template List',
         help='List of all the taxes that have to be installed by the wizard')
-    bank_account_code_prefix = fields.Char(string='Prefix of the bank accounts', oldname="bank_account_code_char")
-    cash_account_code_prefix = fields.Char(string='Prefix of the main cash accounts')
+    bank_account_code_prefix = fields.Char(string='Prefix of the bank accounts', required=True, oldname="bank_account_code_char")
+    cash_account_code_prefix = fields.Char(string='Prefix of the main cash accounts', required=True)
     transfer_account_id = fields.Many2one('account.account.template', string='Transfer Account', required=True,
```

Comment:
this is not required in son l10n_* (mexico for example) even it has been a problem in the past.

Reply:
what do you mean? it has been a problem in the past because it wasn't required or, at the contrary, it is good it is not required now because we had problems in the past (when it was set, I suppose) ?